### ISSUE TITLE
Meaning of lemma notations

### ISSUE THREAD

**Issue by ThomHehl** [openscriptures/morphhb#101]
Hi, totatl Newb to this project and OSIS, but have been a student of Hebrew for years and wanted to dig deeper.

I'm having a look at the lemma notations. I'm going to post Genesis 1:1-2.

        <verse osisID="Gen.1.1">
          <w lemma="b/7225" n="1.0" morph="HR/Ncfsa" id="01xeN">בְּ/רֵאשִׁ֖ית</w>
          <w lemma="1254 a" morph="HVqp3ms" id="01Nvk">בָּרָ֣א</w>
          <w lemma="430" n="1" morph="HNcmpa" id="01TyA">אֱלֹהִ֑ים</w>
          <w lemma="853" morph="HTo" id="01vuQ">אֵ֥ת</w>
          <w lemma="d/8064" n="0.0" morph="HTd/Ncmpa" id="01TSc">הַ/שָּׁמַ֖יִם</w>
          <w lemma="c/853" morph="HC/To" id="01k5P">וְ/אֵ֥ת</w>
          <w lemma="d/776" n="0" morph="HTd/Ncbsa" id="01nPh">הָ/אָֽרֶץ</w><seg type="x-sof-pasuq">׃</seg>
        </verse>
        <verse osisID="Gen.1.2">
          <w lemma="c/d/776" n="1.1.1" morph="HC/Td/Ncbsa" id="01LN3">וְ/הָ/אָ֗רֶץ</w>
          <w lemma="1961" morph="HVqp3fs" id="01Qzf">הָיְתָ֥ה</w>
          <w lemma="8414" n="1.1.0" morph="HNcmsa" id="01aPd">תֹ֨הוּ֙</w>
          <w lemma="c/922" n="1.1" morph="HC/Ncmsa" id="01eYX">וָ/בֹ֔הוּ</w>
          <w lemma="c/2822" n="1.0" morph="HC/Ncmsa" id="01C5U">וְ/חֹ֖שֶׁךְ</w>
          <w lemma="5921 a" morph="HR" id="01qNN">עַל</w><seg type="x-maqqef">־</seg><w lemma="6440" morph="HNcbpc" id="01EVS">פְּנֵ֣י</w>
          <w lemma="8415" n="1" morph="HNcbsa" id="01PB6">תְה֑וֹם</w>
          <w lemma="c/7307" morph="HC/Ncbsc" id="0137c">וְ/ר֣וּחַ</w>
          <w lemma="430" n="0.1" morph="HNcmpa" id="01x9c">אֱלֹהִ֔ים</w>
          <w lemma="7363 b" n="0.0" morph="HVprfsa" id="01yNB">מְרַחֶ֖פֶת</w>
          <w lemma="5921 a" morph="HR" id="0129t">עַל</w><seg type="x-maqqef">־</seg><w lemma="6440" morph="HNcbpc" id="01KZG">פְּנֵ֥י</w>
          <w lemma="d/4325" n="0" morph="HTd/Ncmpa" id="01TZE">הַ/מָּֽיִם</w><seg type="x-sof-pasuq">׃</seg>
        </verse>

It appears that, for example,  1254 is the Strong's number for the Hebrew dictionary.  It appears that b/7225 refers to the strong's number 7225, but what does the b/ mean? I also see c/ and d/.  I assume there's an a/ somewhere or is that the Strong's.

I apologize, but couldn't find any reference info anywhere for this. These would be good to put into the readme.md, which is currently empty.

Thanks and sorry for the newb question.

**Comment by jag3773** [comment]
Hi @ThomHehl , the morphology codes are explained here: https://hb.openscriptures.org/parsing/HebrewMorphologyCodes.html . The first character are the inseparable prefix notations; b (in, by, with) / c (like, as, according to) / d (definite article). Does that help?

**Comment by ThomHehl** [comment]
@jag3773 Yes, thank you so much. This would be good thing to put in the readme as well.

I found definite article on this page, but nothing about the other prepositions.

**Comment by ThomHehl** [comment]
Okay, I have been trying to reconcile the prefixes and suffixes in the lemma codes using the morphology page and I'm lost. I went ahead and created a candidate readme file for this project and I'd like people to review and make whatever changes they wish, but I'd really like the prefixes and suffixes explained there. I have attached it.

Some specific questions.

Are the prefixes "b " and "b/" the same thing? Is it maybe a typo?

I'm assuming that "s/m/" is just "s/" and "m/" stacked. Is that true?

[README.md](https://github.com/openscriptures/morphhb/files/13664575/README.md)

**Comment by ThomHehl** [comment]
@jag3773 Could I get your help on this, please?
